docs/secure.md

```diff
@@ -8,99 +8,68 @@
-
-1. Add the following fields to the MongoDBCommunity resource definition:
+1. Create a TLS-secured MongoDBCommunity resource:
 
-   - `spec.security.tls.enabled`: Encrypts communications using TLS certificates between MongoDB hosts in a replica set and client applications and MongoDB deployments. Set to `true`.
-   - `spec.security.tls.optional`: (**Optional**) Enables the members of the replica set to accept both TLS and non-TLS client connections. Equivalent to setting the MongoDB[`net.tls.mode`](https://www.mongodb.com/docs/manual/reference/configuration-options/#net.tls.mode) setting to `preferSSL`. If omitted, defaults to `false`.
-
-     ---
-     **NOTE**
-
-     When you enable TLS on an existing replica set deployment:
+   ```
+   helm upgrade community-operator mongodb/community-operator --namespace cko-namespace --set resource.tls.useCertManager=true --set createResource=true --set resource.tls.enabled=true
+   ```
 
-     a. Set `spec.security.tls.optional` to `true`.
+  This creates a resource secured with TLS and generates the necessary
+  certificates with `cert-manager` according to the values specified in
+  the `values.yaml` file in the Community Kubernetes Operator 
+  [chart repository](https://github.com/mongodb/helm-charts/tree/main/charts/community-operator).
+  
 
-     b. Apply the configuration to Kubernetes.
 
-     c. Upgrade your existing clients to use TLS.
+1. Test your connection over TLS by 
 
-     d. Remove the `spec.security.tls.optional` field.
+   - Connecting to a `mongod` container using `kubectl`:
 
-     e. Complete the remaining steps in the procedure.
+   ```
+   kubectl exec -it mongodb-replica-set -c mongod -- bash
+   ```
 
-     ---
-   - `spec.security.tls.certificateKeySecretRef.name`: Name of the Kubernetes secret that contains the server certificate and key that you created in the [prerequisites](#prerequisites-1).
-   - `spec.security.tls.caConfigMapRef.name`: Name of the Kubernetes ConfigMap that contains the Certificate Authority certificate used to sign the server certificate that you created in the [prerequisites](#prerequisites-1).
+   Where `mongodb-replica-set` is the name of your MongoDBCommunity resource
 
-   ```yaml
-   apiVersion: mongodbcommunity.mongodb.com/v1
-   kind: MongoDBCommunity
-   metadata:
-     name: example-mongodb
-   spec:
-     members: 3
-     type: ReplicaSet
-     version: "4.2.7"
-     security:
-       tls:
-         enabled: true
-         certificateKeySecretRef:
-           name: <tls-secret-name>
-         caConfigMapRef:
-           name: <tls-ca-configmap-name>
-   ```
+   - Then, use `mongo` to connect over TLS:
 
-1. Apply the configuration to Kubernetes:
    ```
-   kubectl apply -f <example>.yaml --namespace <my-namespace>
+   mongo --tls --tlsCAFile /var/lib/tls/ca/ca.crt --tlsCertificateKeyFile /var/lib/tls/server/*.pem --host <mongodb-replica-set>.<mongodb-replica-set>-svc.<namespace>.svc.cluster.local
```

Comment:
question: will the ca file and the cert key file be mounted to these locations on the replica set pods?